bound/kt/src/main/kotlin/web5/sdk/crypto/verifiers/Ed25519Verifier.kt

```diff
@@ -0,0 +1,27 @@
+package web5.sdk.crypto.verifiers
+
+import web5.sdk.crypto.keys.Jwk
+import web5.sdk.rust.Ed25519Verifier as RustCoreEd25519Verifier
+
+class Ed25519Verifier : Verifier {
+    private val rustCoreVerifier: RustCoreEd25519Verifier
+
+    constructor(privateKey: Jwk) {
+        this.rustCoreVerifier = RustCoreEd25519Verifier(privateKey)
```

Comment:
I think with the latest on `main` you'll need to rebase and then this'll be...

```suggestion
        this.rustCoreVerifier = RustCoreEd25519Verifier(privateKey.rustCoreJwkData)
```